Resources/Prototypes/_DV/Entities/Objects/Misc/sticky_notes.yml

```diff
@@ -0,0 +1,340 @@
+# tags
+
+- type: Tag
+  id: StickyNoteYellow
+
+- type: Tag
+  id: StickyNotePink
+
+- type: Tag
+  id: StickyNoteGreen
+
+- type: Tag
+  id: StickyNoteBlue
+
+# single sticky notes
+
+- type: entity
+  name: sticky note
+  parent: Paper
+  id: PaperStickyNote
+  suffix: yellow
+  description: 'A sticky note, for when you want your reminders to stick'
+  components:
```

Comment:
give it Sticky too so you can stick it to things without throwing

Reply:
this makes sticky notes get stuck in walls without any way to read or retrieve them

Reply:
it also looks like even under accessible circumstances the sticky interactions take all precedence over paper interactions